development/attributions-check.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+yarn attributions:generate
+
+ATTRIBUTIONS_FILE="./attribution.txt"
+
+# check to see if there's changes on the attributions file
+if ! git diff --exit-code "$ATTRIBUTIONS_FILE"; then
+  echo "The set of attributions kept in \`attribution.txt\` are out of date."
+  echo "Run \`yarn attributions:generate\` to regenerate them and commit and push the changes."
+  echo "If you're looking at a pull request, you can also post the comment \`@metamaskbot regenerate-attributions\` and the file will be automatically regenerated for you."
```

Comment:
Oops, looks like this should be:

``` suggestion
  echo "If you're looking at a pull request, you can also post the comment \`@metamaskbot update-attributions\` and the file will be automatically regenerated for you."

Reply:
I haven't paid attention to that neither